data/rulesets.js

```diff
@@ -676,6 +676,18 @@ let BattleFormats = {
 			this.add('rule', 'CFZ Clause: Crystal-free Z-Moves are banned');
 		},
 	},
+    zmoveclause: {
+        effectType: 'ValidatorRule',
+        name: 'Z-Move Clause',
+        desc: "Bans Pok&eacute;mon from holding Z-Crystals",
```

Comment:
... then shouldn't this be called Z-Crystal clause? (Confusingly, the master thread [post](https://www.smogon.com/forums/posts/7397202) says Z-Moves but the [post](https://www.smogon.com/forums/posts/7721063) where the ban was announced says Z-Crystals.)

Reply:
I considered just calling it Z Clause because of this.